frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -53,6 +67,10 @@ export const InfoContent: React.FC<IInfoContent> = ({
   addAssociatedBuildingLink,
 }) => {
   const keycloak = useKeycloakWrapper();
+
+  const canViewProperty =
+    keycloak.hasAgency(propertyInfo?.agencyId as number) ||
```

Comment:
I think even if the user belongs to the agency of the property they still need the view_properties claim.

Reply:
I'll change up the checks here

Reply:
Might want to move this function to the `keycloak` object like @devinleighsmith did with edit